src/index.ts

```diff
@@ -38,6 +46,12 @@ export async function* generatePDF(
 
   for (const file of files) {
     if (file.content) {
+      if (paginationOffset) {
```

Comment:
if you default paginationOffset to 0 in options destructuration, you won't need this if statement.
And since 0 is falsy value, it won't break the ternary condition below.